packages/loader/driver-definitions/src/storage.ts

```diff
@@ -208,19 +221,10 @@ export enum LoaderCachingPolicy {
     Prefetch,
 }
 
-export interface IDocumentServicePolicies {
```

Comment:
the current plan for storage only mode uses this. i'm fine moving caching, but think we should keep this as well

Reply:
If we need policies on service as a whole, we can always add it in future. For not, both policies are for storage part of service only, so it makes sense to move it.